Carwash/Carwash/settings.py

```diff
@@ -146,3 +146,25 @@
         'rest_framework_simplejwt.authentication.JWTAuthentication',
     )
 }
+
+CELERY_BROKER_URL = "redis://redis:6379"
```

Comment:
I recommend to use environment variables for CELERY_BROKER_URL and CELERY_RESULT_BACKEND